en/roadmap.md

```diff
@@ -0,0 +1,33 @@
+---
+title: TiDB Operator Roadmap
+---
+
+# TiDB Operator Roadmap
+
+## Improve the Robustness and Stability of TiDB Operator
```

Comment:
```suggestion
## Improve robustness and stability
```

Reply:
1. We can leave out `TiDB Operator` since this doc is only about TiDB Operator and nothing else.
2. Use sentence case for sub-titles, which means only capitalizing the first letter.